zosia16/settings/common.py

```diff
@@ -133,3 +134,5 @@ def random_string(length=10):
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, '..', 'static'),
 )
+
+MEDIA_ROOT = os.path.join(BASE_DIR, '..', 'media')
```

Comment:
Can you make it `os.path.join(os.environ.get('HOME', '/var/www'), 'media')`? I'd rather not serve static files out of app directory.